etc/org.opencastproject.transcription.microsoftazure.MicrosoftAzureTranscriptionService.cfg

```diff
@@ -0,0 +1,59 @@
+# Change enabled to true to enable this service.
+enabled=false
```

Comment:
Why not enable/disable based on the presence of `subscription.key` and `region`?

Reply:
Because I copied this behaviour from the other services. And maybe there is a situation where you would like to disable the service without having to remove (and later re-enter) your credentials. Not that I can think of one though.